ops/mpt-switch-test/send-txs.sh

```
@@ -0,0 +1,212 @@
+    start)
+        main
+        ;;
+    bg)
+        log_info "Starting in background..."
+        nohup "$0" start > "${SCRIPT_DIR}/.testdata/send-txs.log" 2>&1 &
+        echo $! > "$PID_FILE"
+        log_success "Started (PID: $(cat $PID_FILE))"
+        ;;
+    stop)
+        if [ -f "$PID_FILE" ]; then
+            kill $(cat "$PID_FILE") 2>/dev/null && rm "$PID_FILE"
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)